### ISSUE TITLE
Hide "Allow multiple Values" for child nodes of a nodegroup

### ISSUE THREAD

**Issue by whatisgalen** [archesproject/arches#11048]
If you have a nodegroup, you can enable multiple values for it. If the child nodes aren't on their own nodegroups, they shouldn't have the option to "Allow multiple values". This option is disabled for the children, but I think it would be better to hide the option altogether unless a node becomes its own nodegroup. See video:

https://github.com/archesproject/arches/assets/8366985/952efdec-ebaf-4980-845a-621ee3e149d1

**Comment by chiatt** [comment]
Yeah - that can be confusing, but I don't think it should necessarily be hidden. It might be better to leave it visible, and inform the user that the node 'must be placed in a separate card'. Otherwise there is nothing to indicate that changing the cardinality is even possible.

**Comment by whatisgalen** [comment]
>  Otherwise there is nothing to indicate that changing the cardinality is even possible.

2 thoughts:
- Given that making a node into its own nodegroup is a prerequisite for cardinality change, perhaps it makes sense to communicate that as part of the help directions for "Place node in Separate Card". I get that there would be some redundancy in the directions, but there's no real instruction in the UI that says as much.
- The "disabled" styling really is quite subtle imho; if we want to make it more obvious that making a node into its own nodegroup enables N cardinality, the "Allow Multiple Values" toggle should have a more visually apparent transition from disabled to not disabled

**Comment by chiatt** [comment]
What if we just indented 'Allow Multiple Values' under 'Place Node in Separate Card' so that it is clear that the multiple values is really a dependent condition of being in a separate card.

```
[toggle] Export via Search Results
[toggle] Place Node in Separate Card
        [toggle] Allow Multiple Values
```
